src/edge/io.py

```diff
@@ -39,7 +39,12 @@ def to_gff_file(self, file):
             for annotation in fragment.annotations():
                 # FeatureLocation first bp is AfterPosition, so -1
                 loc = FeatureLocation(annotation.base_first - 1, annotation.base_last)
-                qualifiers = {'name': annotation.feature.name}
+                qualifiers = annotation.feature.qualifiers
+                if "phase" in qualifiers and qualifiers["phase"] is None:
```

Comment:
"phase" is how GFF annotations frame shifts. CDS with phase 1 means annotation is 1 bp frameshift. GFF parser we use parses GFF features into BioPython SeqFeature and stores the "phase" in the qualifiers. Here we guard against manual annotation that set phase to None and change it to 0 instead, because it will be output as None otherwise.

Reply:
Gotcha. That makes sense now! Do you mind adding this as a comment in the code so future people who look at this code can understand the rationale? Thanks!

Reply:
Done